### PR TITLE
Add a default spec for google-fluentd that turns off build_id generation

### PR DESCRIPTION
**Pull request by ekund**
Add a default spec file to use for google-fluentd.  Created by running `omnibus new google-fluentd --rpm-assets` and taking just the spec file. 
Then, added the one line to remove build_id generation.

### CODE REVIEW

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

File: resources/google-fluentd/rpm/spec.erb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Disable any shell actions, replace them with simply 'true'
+%define _build_id_links none
```
[the remark this answers is not in the document]
Comment (ekund) [reply resources/google-fluentd/rpm/spec.erb:1]: Can you please clarify what you mean?  The comment refers to the following line.  What are you expecting?

File: resources/google-fluentd/rpm/spec.erb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+%define __spec_prep_post true
```
[the remark this answers is not in the document]
Comment (ekund) [reply resources/google-fluentd/rpm/spec.erb]: Fixed.

File: resources/google-fluentd/rpm/spec.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (igorpeshansky) [reply resources/google-fluentd/rpm/spec.erb:1]: I meant that the `replace shell actions with 'true'` comment refers to the `%define __spec_… true` lines below, and not to your newly added `%define _build_id_links none` line. But that's been fixed now.

File: resources/google-fluentd/rpm/spec.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+%define _build_id_links none
+<%# pull in the default spec.rb from omnibus %>
```
Comment (igorpeshansky) [inline resources/google-fluentd/rpm/spec.erb:2]: Minor nit: it's `spec.erb`.

Comment (ekund) [reply resources/google-fluentd/rpm/spec.erb]: done.

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**